src/System.Private.Xml/tests/Xslt/XslCompiledTransformApi/XslCompiledTransform.cs

```diff
@@ -2149,6 +2150,7 @@ public void LoadUrl()
         }
 
         //[Variation("Load with \"\\\\\"")]
+        [PlatformSpecific(TestPlatforms.Windows)] //Not a valid path on Unix
```

Comment:
> Not a valid path on Unix

It actually is valid on Unix... and that's the problem for this test, right?  It's expecting an invalid path?